### PR TITLE
Bump Quarkus to 2.12.2.Final

### PR DESCRIPTION
**Pull request by gwenneg**
Based on @pilhuhn's work in #1395. This PR only has one goal: bumping Quarkus.

#1395 is still needed for changes unrelated to the Quarkus bump itself.

### CODE REVIEW

File: engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java [hunk 1]
```diff
@@ -155,20 +154,18 @@ public NotificationHistory doHttpRequest(Notification item, HttpRequest<Buffer>
...
 
-                HttpRequestImpl<Buffer> reqImpl = (HttpRequestImpl<Buffer>) req.getDelegate();
-
                 boolean serverError = false;
                 boolean isEmailEndpoint = item.getEndpoint().getType() == EMAIL_SUBSCRIPTION;
                 boolean shouldResetEndpointServerErrors = false;
                 if (resp.statusCode() >= 200 && resp.statusCode() < 300) {
                     // Accepted
-                    Log.debugf("Webhook request to %s was successful: %d", reqImpl.host(), resp.statusCode());
+                    Log.debugf("Webhook request to %s was successful: %d", url, resp.statusCode());
```
Comment (gwenneg) [inline engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java:162]: Before:
```
2022-09-14 18:24:52,399 DEBUG [com.red.clo.not.pro.web.WebhookTypeProcessor] (executor-thread-43) Webhook request to whatever.com was successful: 200
```

After:
```
2022-09-14 18:24:52,399 DEBUG [com.red.clo.not.pro.web.WebhookTypeProcessor] (executor-thread-43) Webhook request to https://whatever.com/v1/sendEmails was successful: 200
```

File: engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java [hunk 2]
```diff
@@ -226,10 +223,9 @@ public NotificationHistory doHttpRequest(Notification item, HttpRequest<Buffer>
 
                 if (!history.isInvocationResult()) {
                     JsonObject details = new JsonObject();
-                    details.put("url", getCallUrl(reqImpl));
-                    details.put("method", reqImpl.method().name());
+                    details.put("url", url);
```
Comment (gwenneg) [inline engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java:226]: Before:
```
https://whatever.com:443/v1/sendEmails
```

After:
```
https://whatever.com/v1/sendEmails
```

File: engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java [hunk 3]
```diff
@@ -246,30 +242,17 @@ public NotificationHistory doHttpRequest(Notification item, HttpRequest<Buffer>
...
 
-            HttpRequestImpl<Buffer> reqImpl = (HttpRequestImpl<Buffer>) req.getDelegate();
-
             Log.debugf("Failed: %s", e.getMessage());
 
-            // TODO Duplicate code with the error return code part
             JsonObject details = new JsonObject();
-            details.put("url", reqImpl.uri());
-            details.put("method", reqImpl.method());
+            details.put("url", url);
```
Comment (gwenneg) [inline engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java:248]: Before:
```
/v1/sendEmails
```

After:
```
https://whatever.com/v1/sendEmails
```

**Review by josejulio — COMMENTED** [review]
Too late, but lgtm :D

**Pull request merged**